### ISSUE TITLE
comments in .flowconfig

### ISSUE THREAD

**Issue by dckc** [facebook/flow#522]
I like to leave breadcrumbs in files such as .flowconfig, not to mention explaining various bits of config. Is there a comment syntax that I missed? I tried both ; and #.

```
flow-test$ cat .flowconfig 
; .flowconfig -- "start type checking here" sentinel
; http://flowtype.org/docs/new-project.html

flow-test$ flow
Flow server launched for /tmp/flow-test
.flowconfig:1 Unexpected config line not in any section
Could not start flow server!
```

**Comment by gabelevi** [comment]
Nope, no comment syntax yet. Should be easy to add. In any case, I really need to move us away from using a custom format for the configs. https://github.com/facebook/flow/issues/153 tracks that.

**Comment by gabelevi** [comment]
Closed by 5d4ada5e05bf0d344611934bf87fa8571b9b797d
